examples/array-map-symmetry/array_map_symmetry.bal

```diff
@@ -0,0 +1,40 @@
+import ballerina/io;
+
+public function main() {
+    // List constructors are used to construct list values.
```

Comment:
We are missing the information about arrays and maps having a uniform member type.

Reply:
Updated in c9504e7312d263315b1873d5c35d29447168e4fd